myst_nbs/gaussian_processes/GP-Latent.myst.md

```diff
@@ -429,11 +424,24 @@ plt.plot(
 # axis labels and title
 plt.xlabel("X")
 plt.ylabel("True f(x)")
+plt.xlim([0, 12])
 plt.title("Posterior distribution over $f(x)$ at the observed values")
-plt.legend(loc="upper right");
+plt.legend(loc=(0.32, 0.65), frameon=True);
 ```
 
+## Authors
+
+* Created by [Bill Engels](https://github.com/bwengals) in 2017 ([pymc#1674](https://github.com/pymc-devs/pymc/pull/1674)) 
+* Reexecuted by [Colin Caroll](https://github.com/ColCarroll) in 2019 ([pymc#3397](https://github.com/pymc-devs/pymc/pull/3397))
+* Updated for V4 by Bill Engels in June 2022 ([pymc-examples#237](https://github.com/pymc-devs/pymc-examples/pull/237))
+
 ```{code-cell} ipython3
 %load_ext watermark
 %watermark -n -u -v -iv -w
 ```
+
+```{raw-cell}
```

Comment:
This should not be a raw cell either. There should not be any raw cells in the notebook